docs/tre-developers/api.md

```diff
@@ -2,274 +2,181 @@
+      "status": "OK",
+      "message": ""
+    },
+    {
+      "service": "Service Bus",
+      "status": "OK",
+      "message": ""
+    },
+    {
+      "service": "Resource Processor",
+      "status": "Not OK",
+      "message": "Resource Processor is not responding"
+    }
+  ]
+}
+```
 
-The API endpoints documentation and the Swagger UI will be available at [https://localhost:8000/api/docs](https://localhost:8000/api/docs).
+In this case, next step is to look at logs of Resource Processor. See also [Resource Processor docs](resource-processor.md).
 
-### Deploy with Docker
+## Using Swagger UI
 
-You must have Docker and Docker Compose tools installed, and an Azure Cosmos DB configured in `.env` as described above.
+Swagger UI lets you send requests to the API.
 
-Then run:
+To send a request, click on the row with the request, then `Try out`, then `Execute`. See screenshot:
 
-```cmd
-cd api_app
-docker compose up -d app
-```
+[![Swagger UI Send Request](../assets/api_swagger_send_request.png)](../assets/api_swagger_send_request.png)
 
-The API will be available at [https://localhost:8000/api](https://localhost:8000/api) in your browser.
+### Authorization
 
-## Unit tests
+Swagger UI is accessible under `https://${TRE_ID}.${LOCATION}.cloudapp.azure.com/api/docs`.
 
-The unit tests are written with pytest and located in folder `/api_app/tests_ma/`.
+To start using it, click Authorize button, then click Authorize (leave the field `client_secret` empty). See screenshot:
 
-Run all unit tests with the following command in the root folder of the repository:
+[![Swagger UI Authorize](../assets/api_swagger_ui_authorize.png)](../assets/api_swagger_ui_authorize.png)
 
-```cmd
-pytest --ignore=e2e_tests
-```
 
-## API application folder structure
+If you see an error message saying something like:
 
 ```text
-api_app
-├── api              - API implementation
-│   ├── dependencies - Dependencies for routes definition
-│   ├── errors       - Definitions of error handlers
-│   └── routes       - Web routes (API endpoints)
-│
-├── core             - Application configuration, startup events, logging
-│
-├── db               - Database related implementation
-│   ├── migrations   - Manually written alembic migrations
-│   └── repositories - All CRUD features
-│
-├── models           - Pydantic models for this application
-│   ├── domain       - Main models that are used almost everywhere
-│   └── schemas      - Schemas for using in web routes
-│
-├── resources        - Strings that are used e.g., in web responses
-│
-├── services         - Logic that is not only CRUD related
-│
-├── tests_ma         - Unit tests
-│
-└── main.py          - FastAPI application creation and configuration
+The redirect URI 'https://tanyagts8.westeurope.cloudapp.azure.com/api/docs/oauth2-redirect' specified in the request does not match the redirect URIs configured for the application '558602a8-764a-453c-8efa-4dc3ddd61570'.
 ```
 
-## Auth in code
```

Comment:
I removed the following three sections as I felt like they were distracting from the purpose of the doc.
My understanding is that these section are to help understand implementation; could this be better done as comments in the code, closer to implementation?

Reply:
cc @marrobi